azurerm/internal/services/keyvault/key_vault_resource.go

```diff
@@ -195,6 +195,42 @@ func resourceArmKeyVault() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+
+			"private_endpoint_connection": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"private_endpoint_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+
+						"connection_state": {
```

Comment:
this doesn't make sense to expose in Terraform - it's not a field that users can do anything with - can we remove this?